Makefile

```diff
@@ -125,6 +125,9 @@ resetdb: .state/docker-build-base
 initdb: .state/docker-build-base .state/db-populated
 	$(MAKE) reindex
 
+inittuf: .state/docker-build-base
+	docker compose run --rm web rstuf admin ceremony -b -u -f dev/rstuf/bootstrap.json --api-server http://rstuf-api
```

Comment:
add `dev/rstuf/bootstrap.json` to `purge` target.

Reply:
Hi @ewdurbin, I didn't get it. I'm sorry.
Is it to create a `resettuf` to clean up the rstuf db like `resetdb`?
Can you explain?

Reply:
no, see the `purge` task down below. add this file the list of files that are removed when `Make purge` is run.

Reply:
But this file is static. It is not generated or updated dynamically during the process.
The payload uses the root keys and online to initiate the RSTUF.